docs/source/models.md

```diff
@@ -83,7 +107,7 @@ If there is only one _new owner_, the condition will be a single-signature condi
 
 #### Multiple New Owners
 
-If there are multiple _new owners_, we can create a ThresholdCondition requiring a signature from each new owner in order
+If there are multiple _new owners_, they can create a ThresholdCondition requiring a signature from each of them in order
```

Comment:
Yes, in general, a ThresholdCondition can require signatures from only a subset of `new_owners`, but in the example that follows, all of them must sign. There's a comment at the end of this section about the subset option.